src/fragments/lib/datastore/ios/data-access/save-snippet.mdx

```diff
@@ -1,19 +1,20 @@
 <BlockSwitcher>
 
-<Block name="Listener (iOS 11+)">
+<Block name="Async/Await">
 
 ```swift
-Amplify.DataStore.save(
-    Post(title: "My first post",
-         description: "Amplify.DataStore is awesome!",
-         status: .draft)
-) {
-    switch $0 {
-    case .success:
-        print("Created a new post successfully")
-    case .failure(let error):
-        print("Error creating post - \(error.localizedDescription)")
-    }
+do {
+    try await Amplify.DataStore.save(
+        Post(
+            title: "My first post", 
```

Comment:
follow the same pattern as before, ie. first parameter on the same line as the constructor